monitoring-server/cmake/Project.cmake

```diff
@@ -3,7 +3,10 @@ add_library(Core)
 
 target_sources(Core
         PUBLIC
+        include/core/comms/I2C.h
+        include/core/comms/Packet.h
         include/core/defs.h
+        include/core/Device.h
```

Comment:
--- Ignore line 8 included in the context of this comment - can't seem to select lines 6,7 and 9 for one whole comment ---
Are you 100% certain that the order of the header files is correct? I don't know if I am raising a valid point, I have just noticed in the previous commits that you have included each new header file in a new line below the previous header file. Again, I don't know if the order strictly matters, I am just checking with you to make sure you are confident with this.

Reply:
Looks good to me. The order does not matter functionally, it's just a formality. In this case the order chosen is, Folders over Files and A to Z. So you see the `comms` folder before the files, there should be the `events` folder too? Not sure what happened to that!? GitHub says that you commented an outdated version?

Reply:
Oh Nevermind the events, it's because every commit gradually adds the new file to the list of sources as they get added. Anyways the question should have been answered. What do you think?